src/test/java/bg/codexio/springframework/data/jpa/requery/resolver/FilterJsonArgumentResolverTest.java

```diff
@@ -40,6 +46,10 @@ class FilterJsonArgumentResolverTest {
     private CriteriaQuery criteriaQueryMock;
     private CriteriaBuilder mockCriteriaBuilder;
     private Root mockRoot;
+    private static ObjectMapper objectMapperMock;
+    private JsonHttpFilterAdapter httpFilterAdapterMock;
+    private Predicate mockPredicate;
+    private FilterJsonTypeConverter filterJsonTypeConverterMock;
```

Comment:
If this is the unit under test, it shouldn't be a `mock` :)

Reply:
This is the converter that gets the value from the filterRequest and uses the fieldInfo to create the value for the predicate/specification :) The unit under test is the FilterJsonArgumentResolver field with name filterJsonArgumentResolverMock that if needed I can change :)

Reply:
Yes, I tried to find the field with `CTRL+F`, as I saw it before, and found the wrong one. But yes, `filterJsonArgumentResolverMock` is not the correct name for the unit under test. Remove the `mock` suffix :)